docs/articles/other/Your-Expensify-Partner-Manager

```diff
@@ -0,0 +1,33 @@
+---
```

Comment:
This file needs to be named `Your-Expensify-Partner-Manager.md` to show up properly (add the `.md`)